prebid-mobile/pbm-api/iOS/pbm-adunit-ios.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: page_v2
+title: AdUnit - iOS
+description: AdUnit - iOS
+top_nav_section: prebid-mobile
+nav_section: prebid-mobile
+sidebarType: 2
+---
+
+# AdUnit: NSObject
+{: .notoc}
+
+The AdUnit class is the superclass of the [BannerAdUnit](/prebid-mobile/pbm-api/iOS/pbm-bannerad-ios.html) and [InterstitialAdUnit](/prebid-mobile/pbm-api/iOS/pbm-interstitial-ad-ios.html) classes. 
+
+- TOC
+ {:toc}
+
+--- 
+
+## Object 
+
+### AdUnit
+
+Create a new Banner Ad Unit or Interstitial Ad Unit with a Prebid Server configuration ID and a banner size if applicable.
+
+**Parameters**
+
+`configId`:  String containing the Prebid Server configuration ID.
+
+`size:`: CGSize conatining width and height of the AdUnit.
+
+**Properties**
+
+`prebidConfigId`: String containing the Prebid Server configuration ID.
+
+`adSizes`: An Array of CGSizes to be used for AdUnit sizes.
+
+`identifier`: A String Optional, if not nil contains a unique identifier for the AdUnit.
+
+`userKeywords`: Computed property that returns the customerKeywords.
+
+`refreshTime`: A Double Optional used to set the auto refresh time in milliseconds. 
+
+`didReceiveResponse`: A Boolean indicating if the Prebid response was received within the specified time. 
+
+`timeOutSignalSent`: A Boolean to determine if delegate call needs to be made after timeout delegate is sent
+
+`customKeywords <private>`: A Dictionary with each item containing an Array of Strings.
+
+`isInitialCallMade <private>`: A Boolean indicating if a refresh needs to be made even though fetchDemand has not been initialized. 
+
+`adServerObject <private>`: An AnyObject Optional. 
+
+---
+
+## Methods
+
+
+### fetchDemand
+
+Trigger a call to Prebid Server to retrieve demand for this Prebid Mobile ad unit. 
+
+**Parameters**
+
+`adObject`: Object to be passed to `validateAndAttachKeywords` method.
+
+`completion`: Closure which receives one argument, the enum ResultCode. There is no return value. 
+
+### addUserKeyword
+
+Obtains the user keyword and value for targeting of a Prebid Mobile ad unit. If the key already exists the value will be appended to the customKeywords property. No duplicates will be added.
+
+**Parameters**
+
+`key`: A String to be used to check if an existing value exists in the customKeywords property. 
+
+`value`: A String to be appended to the customKeywords property.
+
+### removeUserKeyword
+Remove a key and all its associated values from customKeywords of a given Prebid Mobile ad unit. 
+
+**Parameters**
+
+`forKey`: A string containing the key to remove from customKeywords.
+
+### clearUserKeywords
+Remove all keys and all values from a given Prebid Mobile ad unit.
+
+### setAutoRefreshMillis
```

Comment:
Mention that only PBM refresh is supported so the publisher needs to turnoff adServers refresh manually